org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
This was a suggestion from Davi, a session object only has information about the currently logged in user so it would be redundant to add that prefix.

Also, this service will communicate with a session api in the future and the value will probably come as `contact_id`. We could return whatever name convention we want though. Should I leave it as it is or rename it?